kube-client/src/config/file_config.rs

```diff
@@ -174,6 +199,13 @@ pub struct AuthInfo {
     pub exec: Option<ExecConfig>,
 }
 
+#[cfg(test)]
+impl PartialEq for AuthInfo {
+    fn eq(&self, other: &Self) -> bool {
+        serde_json::to_value(&self).unwrap() == serde_json::to_value(&other).unwrap()
```

Comment:
I think this is fine. It was added recently for `kubeconfig_from_empty_string` test.